nidirect_gp/nidirect_gp.services.yml

```diff
@@ -9,4 +9,4 @@ parameters:
 
   # Regex sourced/tweaked from https://andrewwburns.com/2018/04/10/uk-postcode-validation-regex/.
   # Accepts formats such as: SW1, SW1A, SW1A1AA, SW1A 1AA
-  postcode_matches.uk: '^(([A-Z][0-9]{1,2})|(([A-Z][A-HJ-Y][0-9]{1,2})|(([A-Z][0-9][A-Z])|([A-Z][A-HJ-Y][0-9]?[A-Z]))))((\s?)([0-9][A-Z]{2})?)$'
+  postcode_matches.uk: '^(([A-Z][0-9]{1,2})|(([A-Z][A-HJ-Y][0-9]{1,2})|(([A-Z][0-9][A-Z])|([A-Z][A-HJ-Y][0-9]?[A-Z]))))((\s+)([0-9][A-Z]{2})?)$'
```

Comment:
We need to enforce a space here between the two parts of the postcode.
This is because the geocoder won't work with partial postcodes like 'BT5', a full postcode like 'BT5 8XX' must be supplied.

Reply:
A postcode prefix definitely worked in the past, see https://github.com/dof-dss/nidirect-site-modules/blob/652d19188078595fb83ae8233653bc7eb770a631/nidirect_search/tests/src/Nightwatch/Tests/gpSearchTests.js#L31

Could be geocoder config that needs adjusting to accommodate a certain level of specificity?

Reply:
Good point, I have now added a region setting to the geocoding provider and it works with partial postcodes